src/self_install.rs

```diff
@@ -143,15 +143,15 @@ fn ensure_line(path: &PathBuf, line: &str) -> anyhow::Result<()> {
 
 fn print_post_install_message(settings: &Settings) {
     if cfg!(windows) {
-        print!(r###"# EdgeDB command-line tool is installed now!
+        print!(r###"# The EdgeDB command-line tool has been successfully installed!
```

Comment:
Rust also uses present simple here. This doesn't have to be past perfect passive.